WORKSPACE

```diff
@@ -1,6 +1,6 @@
 git_repository(
     name = "io_bazel_rules_go",
-    commit = "d0142854a22a0dd98306280e897e64086289a0de",
```

Comment:
this bumps bazel up to golang 1.7.5, while our regular builds are still using 1.7.4.